markdown/en/ff-asn.md

```diff
@@ -28,9 +28,9 @@ If you set the ``[filter-style]` attribute properly the `ff-asn-group` acts as d
 ## Using native select box
 
 In version 1.2.7 we introduced native select-box support for `ff-asn-group`.
-Additionally `<option>` element does supports data binding when using
-`<ff-asn-group select-box="true">`. You can now supply your own template
-by annotating it with the data-template attribute.
+Additionally the `<option>` element supports data binding when using
```

Comment:
Let's put a comma after `Additionally`.
https://www.grammarly.com/blog/commas-after-introductory-words/
https://www.searchenginepeople.com/blog/rules-for-comma-use.html